ocdsdata/base.py

```diff
@@ -296,3 +287,10 @@ def run_all(self):
         self.run_fetch()
         self.run_store()
         self.run_check()
+
+
+class SourceSaveUrlResponse:
```

Comment:
I really do not like the name of the class.  I got confused for a while thinking it meant an http response.
My preference would be to call it `SaveUrlResult`.

Also I would consider this class as an inner class of the Source class as it really can't be used elsewhere. 
So it would be called like `return self.SaveUrlResult(...)`, meaning you would not need to import anything apart from Source.